examples/demos/app_push_demo/src/push.c

```diff
@@ -97,6 +97,7 @@ void appMain()
   
   paramVarId_t idPositioningDeck = paramGetVarId("deck", "bcFlow2");
   paramVarId_t idMultiranger = paramGetVarId("deck", "bcMultiranger");
+  paramVarId_t idEmergencyLand = paramGetVarId("deck", "eland"); //Added by M-R-VulkaN
```

Comment:
This should use defined values from the emergency land deck header file.
```suggestion
  paramVarId_t idEmergencyLand = paramGetVarId(ELAND_FLAG_GROUP_NAME, ELAND_FLAG_PARAM_NAME); //Added by M-R-VulkaN
```